mage/query-modules/python/gnn-node-classification.md

```diff
@@ -0,0 +1,343 @@
+---
+id: node-classification-with-gnn
+title: node_classification_with_gnn
+sidebar_label: node_classification_with_gnn
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+export const Highlight = ({children, color}) => (
+<span
+style={{
+      backgroundColor: color,
+      borderRadius: '2px',
+      color: '#fff',
+      padding: '0.2rem',
+    }}>
+{children}
+</span>
+);
+
+[![docs-source](https://img.shields.io/badge/source-node_classification-FB6E00?logo=github&style=for-the-badge)](https://github.com/memgraph/mage/blob/T515-IN-node-classification-with-pyg/python/node_classification.py)
+
+## Abstract
+**Node classification** is the problem of finding out the **right label** for a **node** based on its **neighbors’ labels** and **structure similarities**.
+
+### About the query module
+
+This query module contains all necessary functions you need to train GNN model on Memgraph. 
+
+You can find support for the following features in **node_classification** module:
+- support for both **homogeneous** and **heterogeneous** graphs
+- support for multiple-label and multi-edge-type graphs
+- **no restrictions on dataset size**
+- **multiple model architectures** layers to work on (inductive learning):
+    - **Graph attention with Jumping Knowledge**
+    - multiple versions of **Graph attention network (GAT)**
+    - **GraphSAGE**
+- support for **early stopping**
+- support for different **metrics calculations**
+- **predictions** for specific node
+- **saving** models and **loading** models
+- its applicability to use it as a **recommendation system**
+
+
+If you want to explore our implementation, jump to **[github/memgraph/mage](https://github.com/memgraph/mage)** and find
+`python/node_classification.py`. Feel free to give us a :star: if you like the code. 
+The easiest way to test **node_classification** is by downloading [Memgraph Platform](https://memgraph.com/download)
+and using some of the preloaded datasets in **Memgraph Lab**.
```

Comment:
Maybe rearrange these? It’s easier to try out Lab than to review unfamiliar code

Reply:
agree